cardinal/server/handler/tx.go

```diff
@@ -97,6 +90,23 @@ func PostTransaction(
 	}
 }
 
+// NOTE: duplication for cleaner swagger JSON file
+// PostTransaction godoc
+//
+//	@Summary      Create a persona
+//	@Description  Creates a persona in Cardinal and returns the transaction hash and tick
```

Comment:
"in cardinal" is redundant here; this is a cardinal endpoint, so we already know it's in cardinal. Be consistent whether you want to use -s (`Create` vs `Creates`) or not